api/graphql/schema.graphql

```diff
@@ -135,6 +135,14 @@ interface Notification {
   updatedAt: DateTime @rename(attribute: "updated_at")
 }
 
+type TestNotification implements Notification {
+  id: ID!
+  readAt: DateTime @rename(attribute: "read_at")
+  createdAt: DateTime @rename(attribute: "created_at")
+  updatedAt: DateTime @rename(attribute: "updated_at")
+  data: String @rename(attribute: "data")
+}
+
```

Comment:
Yeah, I am not a fan of this either. I don't think it is a good idea to add to the schema.
Writing up a test schema would be tedious I think? So maybe faker it is.

Reply:
Agreed, turns out faker wasn't really needed not sure what I was thinking with that 😆 

[update tests to not need test notification](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9694/commits/d66dad8a0cf33f250a8fa76b3b3a7e189bf56617)